runtime/zeitgeist/src/parameters.rs

```diff
@@ -365,11 +365,33 @@ parameter_types! {
 }
 
 parameter_type_with_key! {
-    // Well, not every asset is a currency ¯\_(ツ)_/¯
+    // Existential deposits used by orml-tokens.
+    // Only native ZTG and foreign assets should have an existential deposit.
+    // Winning outcome tokens are redeemed completely by the user, losing outcome tokens
+    // are cleaned up automatically.
```

Comment:
I guess we could tolerate that, nevertheless I'd prefer to handle this properly.
I am not able to see a proper solution for that right now. What would be an appropriate existential deposit?
My thoughts go into the direction of automatic resolution in the future. When we have our priority based scheduler (which also handles garbage collection), we can add automatic resolution of outcome tokens as a low priority task to the scheduler. After the last account redeemed we can do some final clean up.

Reply:
My comment here is actually somewhat unjustified. The market accounts are whitelisted from dust collection (with good reason!). So even setting an ED here would not fix the fact that the market account will always survive.

Automatically redeeming the outcome tokens seems like a solution to this, but this means that we definitely need an ED to prevent the scheduler from getting dossed. Since outcome tokens are worth somewhere between nothing and 1 unit of the base asset, I would propose using the same ED as for ZTG/foreign assets.

Reply:
[8f82197](https://github.com/zeitgeistpm/zeitgeist/pull/930/commits/8f82197847a63fe491f6811d60b8780d6a52a933)